src/main/java/com/onarandombox/MultiverseCore/configuration/SpawnLocation.java

```diff
@@ -82,6 +82,24 @@ public Block getBlock() {
         return serialized;
     }
 
+    @Override
+    public boolean equals(Object obj) {
+        if (obj == null) {
+            return false;
+        }
+        if (getClass() != obj.getClass()) {
+            return false;
+        }
+        final Location other = (Location) obj;
+
+        return /* (this.getWorld() == other.getWorld() || (this.getWorld() != null && this.getWorld().equals(other.getWorld())))
```

Comment:
No worries @main--!

I've ported almost all of this over to the new codebase (WorldProperties class) and am making progress. Found a few issues, but nothing insurmountable!

Thanks again for all of your work o/